File: app/src/main/java/com/taxi/friend/taxifriendclient/repositories/RestClient.java
package com.taxi.friend.taxifriendclient.repositories;

import retrofit2.Retrofit;
import retrofit2.converter.jackson.JacksonConverterFactory;

public class RestClient {

    public  static Retrofit createRestClient(){
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("http://192.168.0.9:1323/")
                .addConverterFactory(JacksonConverterFactory.create())
                .build();

        return retrofit;
    }

}

File: app/src/main/java/com/taxi/friend/taxifriendclient/repositories/RestClient.kt
package com.taxi.friend.taxifriendclient.repositories

import retrofit2.Retrofit
import retrofit2.converter.jackson.JacksonConverterFactory

object RestClient {

    fun createRestClient(): Retrofit {

        return Retrofit.Builder()
                .baseUrl("https://igyglwsme3.execute-api.us-east-1.amazonaws.com/prod/")
                .addConverterFactory(JacksonConverterFactory.create())
                .build()
    }

}
